domain/application/service/application.go

```diff
@@ -74,11 +74,20 @@ type ApplicationState interface {
 	// exist.
 	UpsertCloudService(ctx context.Context, appName, providerID string, sAddrs network.SpaceAddresses) error
 
-	// CloudServiceAddresses returns the addresses of the cloud service for the
-	// specified application, returning an error satisfying
-	// [applicationerrors.ApplicationNotFoundError] if the application doesn't
-	// exist.
-	CloudServiceAddresses(ctx context.Context, applicationName string) (network.SpaceAddresses, error)
+	// GetCloudServiceAddresses returns the addresses of the cloud service for the
+	// specified application.
+	GetCloudServiceAddresses(ctx context.Context, appUUID coreapplication.ID) (network.SpaceAddresses, error)
```

Comment:
As far as I can tell, you don't need this. It's only used by the service method of the same name, which has no callers.

In any case, the same purpose is served by `GetUnitPublicAddress`.

Reply:
I need it for wiring the service info `.Addresses()` method that's why nobody is using it yet. It was implemented last pulse and wiring was going to happen this one.

Reply:
But I see your point, we are sure that we'll never return cloud local addresses on `GetUnitPublicAddress` so we can use that method instead. Done.